frontend/src/app/projectmesh/manage-components/manage-mesh-components.component.ts

```diff
@@ -25,8 +28,12 @@ import {ProjectMeshService} from "../project-mesh.service";
 })
 export class ManageMeshComponentsComponent implements OnInit, OnDestroy {
 
+  public defaultConfigurationTemplatesCache: {[key: string]: ConfigurationTemplate[]} = {};
+  public templateVariables: {[key: string]: ValueInfoMap} = {};
+  public ownTemplateVariables: {[key: string]: string} = {};
```

Comment:
Do these have to have this generic object type? Can't we use a `Map<string, ...>` here?